### ISSUE TITLE
[EmbedMaker] Request for edit embeds.

### ISSUE THREAD

**Issue by Ianardo-DiCaprio** [mikeshardmind/SinbadCogs#154]
I think it would be useful to have a `[p]embed edit` command, which could be used something like `[p]embed edit <messageID <newdata>` to then edit an already existing embed, so you don't have to delete the old one and make a new one (which would help for if you need to edit one and want to keep it in the same place in the chat). I also think it would be useful (if possible) so that if the <newdata> isn't put into the command, it somehow sends a message with the current embed data, so you could easily copy and paste it into your command for when you're only making a small change for ease of use.

**Comment by mikeshardmind** [comment]
duplicate of #131 

Already on the planned features list, just waiting on availability to work on it.
